frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -22,6 +22,7 @@ import { PersistentVolumeClaimModel } from '../models';
 
 const { common, ExpandPVC } = Kebab.factory;
 const menuActions = [
+  ...Kebab.getExtensionsActionsForOCS(PersistentVolumeClaimModel),
   ExpandPVC,
```

Comment:
It would be great if we refactor ExpandPVC via your extension point as well.

Reply:
if not please add a todo comment someone from our team can pick it up.

Reply:
I will prefer to target that in separate PR.

Reply:
Why do you need extension for `expand PVC` ? This functionality is generic and available for all.

Reply:
sorry got confused for  `Add capacity`